packages/suite/src/types/utils/yupMethods.d.ts

```diff
@@ -0,0 +1,8 @@
+import * as yup from 'yup'; // eslint-disable-line
```

Comment:
move this to `packages/suite/src/config/suite/yup.ts`, it will work ok and we will have everything in one place

now it's is lost somewhere deep in structure and this must be clearly linked with file `packages/suite/src/config/suite/yup.ts`

Reply:
done